demo.cc

```diff
@@ -632,50 +708,11 @@ class MixpanelUploader final {
               << " \"" << response.body << "\"" << std::endl;
   }
 
-  struct MixpanelQuestionAnsweredEvent {
-    struct Properties {
-      // (reserved) The Mixpanel project token.
-      std::string token;
-
-      // (reserved) The identifier of the user who caused the event to happen.
-      std::string distinct_id;
-
-      // (reserved) The time of the event, in seconds.
-      uint64_t time;
-
-      // Question identifier.
-      schema::QID qid;
-
-      // Answer identifier.
-      schema::ANSWER answer;
+  inline void operator()(schema::UserRecord& u) { PushMixpanelEvent(MixpanelEvent::User(mixpanel_token_, u)); }
```

Comment:
Weird formatting, this goes in one line, the next one in multiple lines. I assume this is what `make indent` did.